wordpress-mobile/WordPress-iOS
# Issue 3496 - Select media from wp media library - Part One

A first step PR to address #3496. I split this fix in multiple PR in order to avoid a complex/lengthy review. 

This PR only allows insertion of site media that is already present on the device. It still doesn't sync with the server all media for the site.

To test it: 

 * Create a new post and insert some images/videos to it using the device media option.
 * After the insertion is done click on insert media again but this time select the site media. 
 * You should see the media that you previously inserted on the post for that blog.
 * You could also go to the feature image option on settings and use site media to select a previous inserted image on the post.

You may see some empty media items on the media picker, this is because we had a bug on the previous version of the app where the media items images where not being saved properly when a app update happen. Just ignore them or if you want  to start clean just delete the app from the simulator or device.

Needs Review: @bummytime
